grapher/scatterCharts/ScatterPlotChart.tsx

```diff
@@ -159,13 +194,22 @@ export class ScatterPlotChart
         if (this.canAddCountry) this.selectionArray.toggleSelection(entityName)
     }
 
-    // Only want to show colors on legend that are actually on the chart right now
-    @computed private get colorsInUse() {
+    // Returns the colors that are used by all points, *across the whole timeline*.
+    // This is why we need the table before the timeline filter is applied.
+    @computed private get colorsInUse(): Color[] {
+        const allValues =
+            this.manager.tableAfterAuthorTimelineAndActiveChartTransformAndPopulationFilter?.get(
```

Comment:
todo post merge: maybe a pattern like `manager.table`, `manager.transformedTable`, and `manager.untransformedTable`, and the latter would just be:

```
Grapher {
computed  get untransformedTable() {
reutrn this.tableAfterAuthorTimelineAndPopulationFilter
}
}
```